audio/whisper/Portfile

```
@@ -0,0 +1,120 @@
+#-*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup   github      1.0
+PortGroup   makefile    1.0
+
+name                whisper
+categories          audio
+github.setup        ggerganov whisper.cpp 1.6.2 v
+github.tarball_from archive
+license             MIT
+maintainers         {ijams.me:nate, @exprez135} \
+                    openmaintainer
+description         Port of OpenAI's Whisper model in C/C++
+long_description    High-performance inference of OpenAI's Whisper automatic speech recognition \
+                    (ASR) model
+notes               "
+Install this port's variants to include models, or download
+your own and use the built-in `-m` feature.
+
+Remember that whisper.cpp only functions on 16-bit WAV
+files. You can use `ffmpeg -i input.mp3 -ar 16000
+output.wav` to convert the file first.
+"
+
+checksums           whisper.cpp-1.6.2.tar.gz \
```

You can use `${version}` here:

```suggestion
checksums           whisper.cpp-${version}.tar.gz \
```